### PR TITLE
(PUP-3816) clean docs and fix bin files - 2.1.x-86

### PR DESCRIPTION
**Pull request by ferventcoder**
This adjusts the bin files to not use hardcoded paths and
removes the contents of the docs folder, something we do
in our vendored ruby to keep the size of the install down.

### CODE REVIEW

File: ruby/bin/catstomp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!C:/ruby-2.1.5-i386-mingw32-base/bin/ruby.exe
+#! ruby
```
Comment (ferventcoder) [inline ruby/bin/catstomp]: @geoffnichols this is what I'm mentioning in ticket [PUP-3816](https://tickets.puppetlabs.com/browse/PUP-3816?focusedCommentId=127277&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-127277)

File: ruby/bin/catstomp.bat [hunk 2]
```diff
@@ -1,6 +1,6 @@
 @ECHO OFF
 IF NOT "%~f0" == "~f0" GOTO :WinNT
-@"ruby.exe" "C:/ruby-2.1.5-i386-mingw32-base/bin/catstomp" %1 %2 %3 %4 %5 %6 %7 %8 %9
+ECHO.This version of Ruby has not been built with support for Windows 95/98/Me.
 GOTO :EOF
 :WinNT
-@"ruby.exe" "%~dpn0" %*
+@"%~dp0ruby.exe" "%~dpn0" %*
```
Comment (ferventcoder) [inline ruby/bin/catstomp.bat]: @geoffnichols and this from ticket [PUP-3816](https://tickets.puppetlabs.com/browse/PUP-3816?focusedCommentId=127277&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-127277)

**Comment by Iristyle** [comment]
Good catches @ferventcoder

**Comment by Iristyle** [comment]
@ferventcoder I think the only remaining question is around `setrbvars.bat`:

https://github.com/puppetlabs/puppet-win32-ruby/blob/2.0.0-x64/ruby/bin/setrbvars.bat

**Comment by ferventcoder** [comment]
@Iristyle right on. Let me take a look at it. If it is not put there by ruby default, why do we put it there?

**Comment by ferventcoder** [comment]
I believe it is put there by Devkit.

**Comment by ferventcoder** [comment]
setrbvars.bat is put there by default.

**Comment by ferventcoder** [comment]
@Iristyle I think we could pull this or I can look at providing those files back to these PRs. And do a comparison on other possibly missing files. I would like to understand why it is missing though.

**Comment by Iristyle** [comment]
https://github.com/oneclick/rubyinstaller/blob/master/resources/installer/setrbvars.bat

**Pull request merged**